content/bn/docs/concepts/policy/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "নীতিমালা"
+weight: 90
+no_list: true
+description: >
+  নীতিগুলির সাথে সুরক্ষা এবং সর্বোত্তম-অনুশীলনগুলি পরিচালনা করুন
+---
+
+<!-- overview -->
+
+কুবারনেটিস নীতিগুলি এমন কনফিগারেশন যা অন্যান্য কনফিগারেশন বা রানটাইম আচরণগুলি পরিচালনা করে। কুবারনেটিস বিভিন্ন ধরণের নীতি সরবরাহ করে নীচে তা বর্ণিত হলো:
+
+<!-- body -->
+
+## এপিআই (API) অবজেক্ট ব্যবহার করে পলিসি প্রয়োগ করুন 
+
+ কিছু API অবজেক্ট নীতি হিসাবে কাজ করে। এখানে কিছু উদাহরণ দেওয়া হল:
+* [নেটওয়ার্ক নীতি](/docs/concepts/services-networking/network-policies/) একটি কাজের চাপের জন্য প্রবেশ এবং প্রস্থানে ট্র্যাফিক সীমাবদ্ধ করতে ব্যবহার করা যেতে পারে।
+* [লিমিটরেঞ্জস](/docs/concepts/policy/limit-range/) বিভিন্ন বস্তুর ধরণের জুড়ে রিসোর্স বরাদ্দের সীমাবদ্ধতা পরিচালনা করে। 
+* [রিসোর্সকোটাস](/docs/concepts/policy/resource-quotas/) একটি জন্য সম্পদ খরচ সীমাবদ্ধ করুন {{< glossary_tooltip text="namespace" term_id="namespace" >}}
+
+## ভর্তি নিয়ন্ত্রক ব্যবহার করে নীতিমালা প্রয়োগ করুন
+
+একটি {{< glossary_tooltip text="admission controller" term_id="admission-controller" >}}
+API সার্ভারে চলে
+এবং API অনুরোধগুলিকে যাচাই বা পরিবর্তন করতে পারে। কিছু ভর্তি নিয়ন্ত্রক নীতি প্রয়োগ করার জন্য কাজ করে।
+উদাহরণস্বরূপ, [অলওয়েজইমেজপুল](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) অ্যাডমিশন কন্ট্রোলার ইমেজ পুল পলিসি 'অলওয়েজ' এ সেট করতে একটি নতুন পড সংশোধন করে।
+
+কুবারনেটিস বেশ কয়েকটি অন্তর্নির্মিত ভর্তি নিয়ামক রয়েছে যা API সার্ভের মাধ্যমে কনফিগারযোগ্য `--enable-admission-plugin` ফ্লাগ। 
+
+ভর্তি নিয়ন্ত্রকদের বিবরণ, উপলব্ধ ভর্তি নিয়ন্ত্রকদের সম্পূর্ণ তালিকা সহ, একটি নিয়োজিত (ডেডিকেটেড) অংশে নথিভুক্ত ( ডকুমেন্ট) করা হয়েছে। 
+
+* [ভর্তি নিয়ন্ত্রকরা](/docs/reference/access-authn-authz/admission-controllers/)
+
+## ভ্যালিডেটিংএডমিশনপলিসি ব্যবহার করে নীতিগুলি প্রয়োগ করুন
+
+ভর্তি নীতিগুলি যাচাই করা কমন এক্সপ্রেশন ল্যাঙ্গুয়েজ (সিইএল) ব্যবহার করে API সার্ভারে কনফিগারযোগ্য বৈধতা চেকগুলি কার্যকর করার অনুমতি দেয়। উদাহরণস্বরূপ, 'সর্বশেষ' চিত্র ট্যাগের ব্যবহার নিষিদ্ধ করতে একটি 'ভ্যালিডেটিং অ্যাডমিশন পলিসি' ব্যবহার করা যেতে পারে।

```

Comment:
`সর্বশেষ`   wrong symbol  .  you have to use  [ ` ]    instead of [ ' ]    

`ভ্যালিডেটিং অ্যাডমিশন পলিসি`   wrong symbol  .  you have to use  [ ` ]    instead of [ ' ]